SmartDeviceLink/SDLLockScreenPresenter.m

```diff
@@ -31,241 +31,173 @@ - (instancetype)init {
     self = [super init];
     if (!self) { return nil; }
 
-    CGRect screenFrame = [[UIScreen mainScreen] bounds];
-    _lockWindow = [[UIWindow alloc] initWithFrame:screenFrame];
-    _lockWindow.backgroundColor = [UIColor clearColor];
-    _screenshotViewController = [[SDLScreenshotViewController alloc] init];
-    _lockWindow.rootViewController = _screenshotViewController;
+    _showLockScreen = NO;
 
     return self;
 }
 
-#pragma mark - Present Lock Window
-
-- (void)present {
-    SDLLogD(@"Trying to present lock screen");
-    dispatch_async(dispatch_get_main_queue(), ^{
-        if (@available(iOS 13.0, *)) {
-            [self sdl_presentIOS13];
-        } else {
-            [self sdl_presentIOS12];
-        }
-    });
-}
+- (void)stop {
+    self.showLockScreen = NO;
 
-- (void)sdl_presentIOS12 {
-    if (self.lockWindow.isKeyWindow) {
-        SDLLogW(@"Attempted to present lock window when it is already presented");
+    if (!self.lockWindow) {
```

Comment:
```suggestion
    if (self.lockWindow == nil) {
```

In general, I prefer when checking nil to do the explicit check. This isn't entirely consistent across the project, but I think it reads more clearly.

Reply:
Fixed.